index.html

```diff
@@ -18,6 +18,11 @@
         <div class="logo">
           <a href="#"><img src="assets/images/mobile/My-Logo.svg" alt="Logo"></a>
         </div>
+        <ul class="nav-item">
+          <li class="nav-item1"><a href="#portfolio">Portfolio</a></li>
+          <li class="nav-item2"><a href="#about">About</a></li>
+          <li class="nav-item3"><a href="#contact">Contact</a></li>
+        </ul>
         <div class="menu-icon">
           <a href="#"><img src="assets/images/mobile/Icon-Menu.svg" alt="Menue"></a>
         </div>
```

Comment:
Please follow the design for the navigation bar. Kindly see the screenshot below and make changes accordingly.
![nav](https://user-images.githubusercontent.com/35504149/135630403-49733372-9ead-46ad-9851-0d21f441fbf5.png)

Reply:
This screenshot seems to be for mobile device,s not desktop

![image](https://user-images.githubusercontent.com/68148157/135676942-f71e0281-1e3d-49e6-9a22-3ff450afae75.png)


I need clarification. Thank you